src/libexec/zmsyslogsetup

```diff
@@ -313,7 +313,7 @@ sub updateSyslog {
     while (<>) {
       if ( $rsyslog == 1 ) {
         s/syslogd\*.pid/rsyslogd.pid/g;
-        if ($platform eq "RHEL7_64") {
+        if ($platform eq "RHEL7_64"  || $platform eq "RHEL8_64") {
```

Comment:
do we need this fix for RHEL9 as well, as we are adding that support in https://github.com/Zimbra/zm-core-utils/pull/141/files

Reply:
yes, @Gopal-Moolchandani please add it for RHEL9 as well.

Reply:
sure